nomad/deploymentwatcher/deployment_watcher.go

```diff
@@ -163,9 +192,10 @@ func (w *deploymentWatcher) SetAllocHealth(
 	// Create the request
 	areq := &structs.ApplyDeploymentAllocHealthRequest{
 		DeploymentAllocHealthRequest: *req,
-		Eval:             w.getEval(),
-		DeploymentUpdate: u,
-		Job:              j,
+		Timestamp:                    time.Now(),
+		Eval:                         w.getEval(),
+		DeploymentUpdate:             u,
```

Comment:
`u` and `j` are pretty terrible names :) I know these are inherited from past code but if it is easy to update them to something more descriptive that would be excellent

Reply:
Agree in this case that they can be better but this is very much a go style: https://github.com/golang/go/wiki/CodeReviewComments#variable-names